src/Orckestra.Composer.Website/App_Code/OrdersHelper.cshtml

```diff
@@ -0,0 +1,78 @@
+@using Composite.AspNet.Razor
+
+@helper CompactOrderDetails(bool IsPastOrders = false)
+{
+    <div v-for="order in Orders" class="card mb-2">
+        <div class="card-header">
+            <div class="row">
+                <div class="col-sm-8">
+                    <h5 class="m-0"> @Html.Localized("MyAccount", "L_OrderedOn", "{{order.OrderInfos.OrderDate}}")</h5>
+                </div>
+                <div class="col-sm-4 text-sm-right">
+                    <a v-bind:href="order.Url" class="cta text-dark font-weight-normal">
+                        @Html.Localize("MyAccount", "H_OrderDetail")
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="card-body">
+            <div class="row">
+                <div class="col-sm-8">
+                    @Html.Localized("MyAccount", "L_OrderNumber", "{{order.OrderInfos.OrderNumber}}")
+                    @if (!IsPastOrders)
+                    {
+                        <br />
+                        <span class="text-aldi-gray-error" v-if="order.ShipmentSummaries[0].ScheduledShipDate">
+                            @Html.Localized("MyAccount", "L_ScheduledShipping", "{{order.ShipmentSummaries[0].ScheduledShipDate}}")
+                        </span>
+                        <div v-if="order.ShipmentSummaries[0].TrackingInfo">
+                            <a v-if="order.ShipmentSummaries[0].TrackingInfo" v-bind:href="order.ShipmentSummaries[0].TrackingInfo.Url">
+                                @Html.Localize("MyAccount", "H_OrderTracking")
+                            </a>
+                        </div>
```

Comment:
need check if order.ShipmentSummaries[0] exist for this part of code
something like v-if="order.ShipmentSummaries.length"